tt_eager/tensor/tensor_utils.cpp

```diff
@@ -204,17 +204,15 @@ const Shape infer_dims_for_reshape(int N, int C, int H, int W, uint32_t old_volu
 
   bool is_device_tensor(const Tensor& tensor) { return tensor.storage_type() == StorageType::DEVICE; }
 
-Tensor get_device_tensor(const Device* device, const Tensor& multi_device_tensor) {
+Tensor get_device_tensor(Device* device, const Tensor& multi_device_tensor) {
```

Comment:
did we need to remove the const?

Reply:
We needed to remove const, since we're doing map accesses with the device ptr and apparently these can't work on const objects.